gradle.properties

```diff
@@ -10,5 +10,5 @@
 
 commonsCLIVersion=1.3.1
 commonsCSVVersion=1.5
-pravegaVersion=0.5.0
+pravegaVersion=0.6.0-2364.fb3423a-SNAPSHOT
```

Comment:
We may need to add the JFrog repo to get these snapshot build: https://oss.jfrog.org/artifactory/jfrog-dependencies
You can add this in the `build.gradle` in the `repositories` section:
```
maven {                                                       
    url 'https://oss.jfrog.org/artifactory/jfrog-dependencies'
}                                                             
```

Reply:
Fixed.